app/src/main/java/org/glucosio/android/db/DatabaseHandler.java

```diff
@@ -310,15 +307,14 @@ public GlucoseReading getGlucoseReadingById(long id) {
         return idArray;
     }
 
-    public ArrayList<Integer> getGlucoseReadingAsArray() {
+    public List<Double> getGlucoseReadingAsArray() {
```

Comment:
The method name does not reflect the return type.

Reply:
@WBertan is right, before change the name reflected the return type, now it is misleading, maybe `getGlucoseReadingAsList`?

Reply:
Renamed and updated a little bit